### PR TITLE
Created the event model as well as corresponding tests and migrations.

### PR DESCRIPTION
**Pull request by jvdoorn**
As specified in #3 I have created the Event model which also validates that the `start_time` is before the `end_time`.

### CODE REVIEW

**Comment by jvdoorn** [comment]
I'll have a look at why those tests failed later today and make an attempt to fix them.

File: hivemind/modules/api/tests/test_signals.py [hunk 1]
```diff
@@ -31,3 +34,35 @@ def test_user_creates_profile(self):
         self.assertEqual(profile.first_name, user.first_name)
         self.assertEqual(profile.last_name, user.last_name)
         self.assertEqual(profile.email, user.email)
+
+    def test_event_clean(self):
```
Comment (DavisRayM) [inline hivemind/modules/api/tests/test_signals.py]: Hey 😄  first off, Thank you very much for the contribution. Secondly, could you move this test into a TestCase of its own preferably the Event Model Tests as it doesn't truly fit in as a Signal. Other than that, thank you for contributing.

**Review by DavisRayM — CHANGES_REQUESTED** [review]

File: hivemind/modules/api/models/event.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    start_time = models.DateTimeField(
+        'Start Time',
+        help_text='This is the time when the event starts.'
+    )
+    end_time = models.DateTimeField(
+        'End Time',
+        help_text='This is the time when the event ends.'
+    )
+
```
Comment (DavisRayM) [inline hivemind/modules/api/models/event.py:32]: Please add a `created` and `modified` field. The `created` field will store the Date & Time at which the event is created while the `modified` field will store the last Date & time the Event was modified.

**Comment by jvdoorn** [comment]
If a lot of models are going to use `modified` and `created` fields we might want to create an abstract model that we can easily extend, see this [StackOverflow answer](https://stackoverflow.com/a/11031907/8150085) for more details on what I mean.

**Comment by DavisRayM** [comment]
Seems like a good idea. Would you like to add it in this issue, by creating a ‘BaseModel’ class for now. Containing both the ‘created’ and ‘modifed’ fields

**Comment by jvdoorn** [comment]
Sure thing, on it.

File: hivemind/modules/api/migrations/0004_auto_20181003_1015.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.1.1 on 2018-10-03 08:15
+
```
Comment (DavisRayM) [inline hivemind/modules/api/migrations/0004_auto_20181003_1015.py:2]: Not quite comfortable with having multiple Migrations for a new feature as it hasn't been implemented on any thing yet. How about we have one migration for this new model ?

**Comment by DavisRayM** [comment]
Looks good to me so far. Will merge this into master after a bit. Just for the future though while making a PR from a fork please make sure that the branch you are working on within the fork isn't master.

**Pull request merged**